docs/zephyr-integration.md

```diff
@@ -0,0 +1,493 @@
+
+Manage test case relationships and versioning.
+
+#### `zephyr_get_test_case_links`
+
+Get all links for a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+
+#### `zephyr_create_test_case_web_link`
+
+Create a web link for a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `webLinkData` (required): Web link data with URL and optional description
+  - `url` (required): Web link URL (must start with http:// or https://)
+  - `description` (optional): Optional description for the web link
+
+#### `zephyr_list_test_case_versions`
+
+List all versions of a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `maxResults` (optional): Maximum number of results (default: 10, max: 1000)
+
+#### `zephyr_get_test_case_version`
+
+Get a specific version of a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `version` (required): Version number of the test case to retrieve
+
+### Test Scripts and Steps
+
+Retrieve test case implementation details.
+
+#### `zephyr_get_test_case_test_script`
+
+Get the test script for a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+
+#### `zephyr_get_test_case_test_steps`
+
+Get test steps for a test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `maxResults` (optional): Maximum number of results (default: 10, max: 1000)
+- `startAt` (optional): Zero-indexed starting position (default: 0)
+
+
+## Common Workflows
+
+### Creating Test Cases
+
+Complete workflow for creating comprehensive test cases:
+
+```json
+// 1. Create the test case
+{
+  "tool": "zephyr_create_test_case",
+  "parameters": {
+    "testCaseData": {
+      "name": "User Registration Validation",
+      "objective": "Verify new user registration process",
+      "projectKey": "ECOM",
+      "folderId": 456,
+      "priorityName": "High"
+    }
+  }
+}
+
+// 2. Add detailed test steps
+{
+  "tool": "zephyr_add_test_steps",
+  "parameters": {
+    "testCaseKey": "ECOM-T789",
+    "testSteps": [
+      {
+        "description": "Navigate to registration page",
+        "expectedResult": "Registration form displays all required fields"
+      },
+      {
+        "description": "Fill form with valid data",
+        "expectedResult": "Form accepts input without validation errors",
+        "testData": "email: test@example.com, password: SecurePass123"
+      },
+      {
+        "description": "Submit registration",
+        "expectedResult": "User account created and confirmation email sent"
+      }
+    ]
+  }
+}
+
+// 3. Link to requirement issue
+{
+  "tool": "zephyr_link_test_case_to_issue",
+  "parameters": {
+    "testCaseKey": "ECOM-T789",
+    "issueKey": "ECOM-REQ-123"
```

Comment:
```suggestion
    "issueKey": "ECOM-123"
```
ECOM-REQ is not a valid projectKey

Reply:
but linking test case to issue should be issueId:
https://support.smartbear.com/zephyr-scale-cloud/api-docs/#tag/Test-Cases/operation/createTestCaseIssueLink